types/micro/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "socket.io": ">=3.0.0"
```

Comment:
See above, this is used for testing only and should not be made a hard, consider replacing socket.io from the test as a whole.

Reply:
Thanks for your comment. The way I've done it is the way suggested in the README.

Are you the maintainer of the hapi types? Feel free to suggest how to get rid of this test dependency. The tests are very complicated to me, I'm not sure how I would go about even starting to change them.

Reply:
Also, the test code is an exact copy from the hapi README:
https://github.com/hapijs/hapi/blob/master/API.md#-serverlistener